evaluation/SWE-bench/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN conda env create -f environment.yml
 
 # Add commands
 COPY ./commands.sh .
-RUN source commands.sh
+RUN . commands.sh
```

Comment:
Any reason for this? I find `source` more readable

Reply:
I ended up tripping over this in another PR 😄 turns out `source` isn't available on the machine